Assets/Mirror/Tests/Editor/MessageInheritanceTest.cs

```diff
@@ -1,127 +0,0 @@
-using NUnit.Framework;
-
-namespace Mirror.Tests.MessageInheritance
```

Comment:
We should keep these Inheritance tests. Some users use Inheritance so we should still support it as an option.